src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultHtmlFactsService.cs

```diff
@@ -10,29 +10,34 @@ namespace Microsoft.VisualStudio.Editor.Razor;
 
 internal class DefaultHtmlFactsService : HtmlFactsService
 {
-    public override bool TryGetElementInfo(SyntaxNode element, out SyntaxToken containingTagNameToken, out SyntaxList<RazorSyntaxNode> attributeNodes)
+    public override bool TryGetElementInfo(SyntaxNode element, out SyntaxToken containingTagNameToken, out SyntaxList<RazorSyntaxNode> attributeNodes, out SyntaxToken lastTokenInsideBody)
```

Comment:
Consider renaming `lastTokenInsideBody` to `forwardSlashOrCloseAngleToken` (or a better name you come up with) to make it clear at call sites that it won't return whitespace or attributes. eg, with a tag of `<goo minimized>` its not possible to tell from the parameter name whether it would return the minimized attribute name.

Reply:
Or in other words, given `<goo>` the "last" token in the "body" is the `>`, but in `<goo/>` its the `/`, even though `>` is clearly after it 😛

Reply:
Done, thanks for the suggestion.